plugins/continuity.js

```diff
@@ -1383,6 +1383,17 @@
 						elements[i].src ||
 						(typeof elements[i].getAttribute === "function" && elements[i].getAttribute("xlink:href"));
 
+					// if src if not defined, look for it in css background image
+					if (!src) {
+						if (typeof BOOMR.window.getComputedStyle === "function") {
+							var bgStyle = BOOMR.window.getComputedStyle(elements[i]).getPropertyValue('background');
```

Comment:
Last thing, can we do `null` checks on the return values of both `getComputedStyle()` and `getPropertyValue()`?

I see some [random notes about them returning `null`](https://www.fxsitecompat.dev/en-CA/docs/2018/getcomputedstyle-no-longer-returns-null-when-style-can-t-be-retrieved/) in [some cases](http://forums.mozillazine.org/viewtopic.php?t=647310) on the web.

Also, please run `grunt lint` -- there's a few small lint fixes needed.

That should be it!  Test looks great, thanks.

Reply:
Changes made.